File: Assignment6/Program3.c
// 3.Write a program which accept number from user and return the count of digits in between 3 and 7. 
// Input :  2395  Output :  1 
// Input :  1018 Output :  0 
// Input :  4521 Output :  2 
// Input :  9922 Output :  0

#include<stdio.h>
int CountRange(int);
int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter number");
    scanf("%d",&iValue);

    iRet = CountRange(iValue);
    printf("%d ",iRet);

    return 0;
}

int CountRange(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;
    if(iNo < 0)
    {
        iNo = -iNo;
    }

    while(iNo > 0)
    {
        iDigit = iNo % 10;
        iNo =iNo /10;
        if(iDigit > 3 && iDigit < 7)
        {
            iCnt++;
        }
    }
    return iCnt;
}

File: Assignment33/Program3.java
// 3.Write a program which accept number from user and return the count of digits in 
// between 3 and 7. 
// Input :  2395  Output :  1 
// Input :  1018 Output :  0 
// Input :  4521 Output :  2 
// Input :  9922 Output :  0 

import java.util.*;

class Program3
{
	public static void main(String[] args) 
	{
		Scanner sobj = new Scanner(System.in);
		int iValue = 0,iRet = 0;

		System.out.println("Enter number");
		iValue = sobj.nextInt();

		Digit dobj = new Digit();
		iRet = dobj.CountRange(iValue);

		System.out.println("Range is :"+iRet);
	}	
}

class Digit 
{  
	public int CountRange(int iNo)  
	{ 
		int iDigit = 0,iCnt = 0;

		while(iNo != 0)
		{
			iDigit = iNo % 10;
			if(iDigit > 3 && iDigit < 7)
			{
				iCnt++;
			}
			iNo = iNo / 10;
		}
		return iCnt;
	} 
}